doc/yaml_configuration.md

```diff
@@ -646,18 +688,7 @@ setup-info:
         url: "https://example.com/ghc-7.10.2-i386-unknown-mingw32-foo.tar.xz"
 ```
 
-Or you can point to external setup-info:
-
-```yaml
-setup-info: "https://example.com/my-stack-setup-info.yaml"
-```
-
-This may be either URL or (since 1.2.0) absolute file path.
-
-Note that this **adds** the specified setup info metadata to the default.
-If you need to **replace** it, use the `stack --setup-info-yaml` command-line
-argument instead.  The default setup metadata is in
-[stack-setup-2.yaml](https://github.com/commercialhaskell/stackage-content/raw/master/stack/stack-setup-2.yaml).
+Note that specifying this config *does not* the default `stack-setup-2.yaml` from being consulted as a fallback.
```

Comment:
I think there's a typo in this sentence, I'm not sure how it's supposed to read.